src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -195,6 +196,8 @@ const WorkflowView = _.flow(
       this.setState({
         isFreshData: true, savedConfig: config, modifiedConfig: config,
         entityMetadata, inputsOutputs,
+        firecloudRoot: await Config.getFirecloudUrlRoot(),
+        dockstoreRoot: await Config.getDockstoreUrlRoot(),
```

Comment:
FYI: This is not actually a problem here, since the config will already be loaded by the time we get here, but generally it's good to think about parallelizing calls like this.